firmware/openbot_nano/openbot_nano.ino

```diff
@@ -251,10 +341,12 @@ int ctrl_min = (int) 255.0 * VOLTAGE_MIN / VOLTAGE_MAX;
 #endif
 
 #if HAS_SONAR
+#if (OPENBOT != RTR_520)
 #include "PinChangeInterrupt.h"
+#endif
 //Sonar sensor
 const float US_TO_CM = 0.01715;              //cm/uS -> (343 * 100 / 1000000) / 2;
-const unsigned int MAX_SONAR_DISTANCE = -1;  //cm
+const unsigned int MAX_SONAR_DISTANCE = 300;  //cm
 const unsigned int MAX_SONAR_TIME = MAX_SONAR_DISTANCE * 2 * 10 / 343 + 1;
```

Comment:
As discussed with Matthias, an alternative solution consists in changing this line to:
```
const unsigned long MAX_SONAR_TIME = (long)(MAX_SONAR_DISTANCE * 2 * 10 / 343 + 1);
```

Reply:
Updated as discussed.